app/javascript/mastodon/features/notifications/components/column_settings.js

```diff
@@ -30,8 +35,40 @@ export default class ColumnSettings extends React.PureComponent {
     const showPushSettings = pushSettings.get('browserSupport') && pushSettings.get('isSubscribed');
     const pushStr = showPushSettings && <FormattedMessage id='notifications.column_settings.push' defaultMessage='Push notifications' />;
 
+    const settingsIssues = [];
+
+    if (alertsEnabled && browserSupport && browserPermission !== 'granted') {
+      if (browserPermission === 'denied') {
+        settingsIssues.push(
+          <button
+            className='text-btn column-header__issue-btn'
+            tabIndex='0'
+            onClick={onRequestNotificationPermission}
+          >
+            <Icon id='info-circle' /> <FormattedMessage id='notifications.permission_denied' defaultMessage='Mastodon cannot show notifications because the permission has been denied' />
```

Comment:
Maybe the info-circle is not the ideal icon here, it somehow "bites" with the red color: The red color suggests it's important, the information-icon suggests it's not that important (warning vs informational)

exclamation-circle may be more appropiate. 

What do you think?

Reply:
Good point. Changed it!
![image](https://user-images.githubusercontent.com/384364/80302698-ea670980-87ab-11ea-87f2-9bfe35bcfdb7.png)